internal/log_analysis/log_processor/parsers/awslogs/alb.go

```
@@ -87,28 +88,24 @@ func (p *ALBParser) New() parsers.LogParser {
 }
 
 // Parse returns the parsed events or nil if parsing failed
-func (p *ALBParser) Parse(log string) []*parsers.PantherLog {
+func (p *ALBParser) Parse(log string) ([]*parsers.PantherLog, error) {
 	record, err := p.CSVReader.Parse(log)
 	if err != nil {
-		zap.L().Debug("failed to parse the log as csv")
-		return nil
+		return nil, err
 	}
 
 	if len(record) < albMinNumberOfColumns {
-		zap.L().Debug("failed to parse the log as csv (wrong number of columns)")
-		return nil
+		return nil, err
```

`err` will be nil. You need to return an error with custom message.